lib/apiservers/portlayer/swagger.json

```diff
@@ -2408,6 +2686,138 @@
 					}
 				}
 			}
+		},
+		"LoggingJoinConfig": {
+			"type": "object",
+			"required": [
+				"handle"
+			],
+			"properties": {
+				"handle": {
+					"type": "object"
```

Comment:
Actually I'll introduce them back and instead and do that exec.GetHandle surgery in a different PR. They are related but Handle refactoring has nothing to do with 2 new subsystems.